cmd/tempo-cli/cmd-rewrite-blocks_test.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/parquet-go/parquet-go"
+	"github.com/stretchr/testify/require"
+
+	tempo_io "github.com/grafana/tempo/pkg/io"
+	"github.com/grafana/tempo/pkg/parquetquery"
+	"github.com/grafana/tempo/tempodb/backend"
+	"github.com/grafana/tempo/tempodb/backend/local"
+	"github.com/grafana/tempo/tempodb/encoding/common"
+	"github.com/grafana/tempo/tempodb/encoding/vparquet4"
+)
+
+func TestDropTraceCmd(t *testing.T) {
+	for s := 2; s < 10; s++ {
+		s := s
+		t.Run(fmt.Sprintf("drop every %d trace", s), func(t *testing.T) {
+			tempDir := t.TempDir()
+			err := os.CopyFS(tempDir, os.DirFS("./test-data2"))
```

Comment:
I see that only `test-data2` is used, can we cleanup test-data dirs, and only keep one dir, and name it `testdata`.

the name `testdata` is the convention for storing test data.

Reply:
we prefer to not checkin blocks and generate it for the testing.

see `makeBackendBlockWithTraces‎` for reference https://github.com/grafana/tempo/blob/403fdcf14e595281377a23b34a6c61612a6d5216/tempodb/encoding/vparquet4/block_search_test.go#L252